s3/pkg/model/xmlstruct.go

```diff
@@ -121,4 +121,14 @@ type StorageClass struct {
 type ListStorageClasses struct {
 	Xmlns       string `xml:"xmlns,attr"`
 	Classes     []StorageClass `xml:"Class"`
-}
\ No newline at end of file
+}
+
+type LoadObjectsReq struct {
+	Backend string `xml:"Backend"`  // backend name
+	Prefix  string `xml:"Prefix"`   // load those objects with this prefix
+}
+
+type LoadObjectsResponse struct {
```

Comment:
Please use goimports to format the code.

Reply:
Done